### ISSUE TITLE
pypi stripe 2.0 package can not be imported

### ISSUE THREAD

**Issue by btel** [stripe/stripe-python#444]
Installing stripe 2.0 using pip in python 3.5 gives an package that can not be imported:

```
pip install stripe
python -c 'import stripe'
ImportError: No module named 'stripe'
```

**Comment by ob-stripe** [comment]
Huh, weird. I can reproduce, but I'm really not sure what's happening. Installing manually via `python setup.py install`, or installing the wheel via `python setup.py bdist_wheel --universal; pip install dist/stripe-2.0.0-py2.py3-none-any.whl` both work as expected. I'll keep looking for the issue.

**Comment by ob-stripe** [comment]
Looks like something's wrong with the wheel that was uploaded to PyPI.

**Comment by btel** [comment]
It seems that only the metadata have been packaged, but not the python files.

**Comment by ob-stripe** [comment]
Yes, looks like our release tooling does not like the changes to `setup.py`.

**Comment by ob-stripe** [comment]
Alright, I think I've found the issue and prepared a fix in #445. We should release a fixed version later today. Thanks for the report @btel and sorry for the inconvenience!

**Comment by ob-stripe** [comment]
Alright, this should be fixed in 2.0.1. Let me know if you're still having trouble!

**Comment by btel** [comment]
It seems to be working. Thanks for fixing and fast response!
